### PR TITLE
Allow Terraform 1.0.x to use this module

### PR DESCRIPTION
**Pull request by kevcube**
Hey, needed to run this on 1.0.2 and it worked fine. PR to change `versions.tf` to allow any terraform 1.0.x to use this module.

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 0.12.20, < 0.16"
+  required_version = ">= 0.12.20, ~> 1.0.0"
```
Comment (mariux) [inline versions.tf:2]: this will limit it to `1.0.x` and not add support.

```suggestion
  required_version = ">= 0.12.20, < 1.1.0"
```

**Review by mariux — CHANGES_REQUESTED** [review]
Thanks for your feedback and contribution. we would need to update some more files in order to release a version supporting this as tests also need to run with 1.0.x in the CI from now. 

We can still merge it and use this as a base so people can use a git reference to the commit-sha when using the module with 1.0.x already.

File: versions.tf (hunk 1 above)
Comment (kevcube) [reply versions.tf:2]: I have tested this and it works with Terraform 1.0.2

See https://www.terraform.io/docs/language/expressions/version-constraints.html#gt--1

Comment (kevcube) [reply versions.tf:2]: Ah I understand the point you are making, this negates the `>=0.12.20`

**Review by mariux — APPROVED** [review]

**Comment by mariux** [comment]
merging is blocked as we require a pr brnach to be up to date with latest base branch... could you please merge in latest base branch?

**Comment by kevcube** [comment]
@mariux done 😄

**Pull request merged**